packages/kbn-visualization-utils/src/debounced_value.test.ts

```diff
@@ -9,23 +9,24 @@
 import { act, renderHook } from '@testing-library/react-hooks';
 import { useDebouncedValue } from './debounced_value';
 
-jest.mock('lodash', () => {
-  const original = jest.requireActual('lodash');
+describe('useDebouncedValue', () => {
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
 
-  return {
-    ...original,
-    debounce: (fn: unknown) => fn,
-  };
-});
```

Comment:
Your changes here are definitely a better way to test this. The JSDocs of the `useDebouncedValue` function [here](https://github.com/elastic/kibana/pull/187143/files#diff-6faf34257d489dcc95b58074a1e75b4db897351ba0b668ce59561bc0321b0417R19) reference this spec as an example how to mock the `lodash.debounce` which is no longer the case, so may want to remove that comment.

```diff
- * When testing this function mock the "debounce" function in lodash (see this module test for an example)
+ * When testing this function mock the "debounce" function in lodash
```